keyboard_to_ackermann_node:
  ros__parameters:
    ackermann_topic: "/ackermann_cmd"
    frame_id: "base_link"

    forward_speed_mps: 0.2
    reverse_speed_mps: 0.2
    max_steer_rad: 0.24
    steer_step_rad: 0.02

    publish_rate_hz: 30.0
    throttle_timeout_ms: 250
    debug_print: false
